packages/forms-web-app/tests/unit/routes/index.test.js

```diff
@@ -21,6 +21,7 @@ describe('routes/index', () => {
     expect(use).toHaveBeenCalledWith('/cookies', cookieRouter);
     expect(use).toHaveBeenCalledWith('/appellant-submission', appellantSubmissionRouter);
     expect(use).toHaveBeenCalledWith('/eligibility', eligibilityRouter);
-    expect(use.mock.calls.length).toBe(5);
+    expect(use).toHaveBeenCalledWith('/your-planning-appeal', eligibilityRouter);
```

Comment:
highlights the tests here are less than stellar here:

`expect(use).toHaveBeenCalledWith('/your-planning-appeal', yourPlanniningAppealRouter);`